### DISCUSSION TITLE
is mithril js dead?

### DISCUSSION THREAD

**Discussion by ghost** [MithrilJS/mithril.js#2697] [General]
it's been so long that i haven't seen any commits in this repo. is it dead or it's as stable as a rock that don't need any commits or updation?

**Comment by panoply** [comment]
Stable as rock fam. Do not stress.

**Comment by kevinfiol** [comment]
The framework itself is stable as a rock as @panoply [mentioned], although the docs are in dire need of some care.

**Comment by tbreuss** [comment]
Absolutely not dead, but rather very alive. Still my favorite JS framework with a very friendly community.

To quote @ACXgit from [gitter](https://gitter.im/mithriljs/mithril.js):

> Still using Mithril every single day here, and not missing anything!

There is nothing more to say 😉

**Comment by StephanHoyer** [comment]
Same here. We run a massive code base with no issues regarding the framework. Not planning to switch.

Although I would love to pay a full time dev to clean up long outstanding issues.

**Reply by StephanHoyer** [reply]
Just to be clear, I would love to pay, but I can't 😂

**Reply by ghost** [reply]
Pay me and I look into it :) I like mithril to but it has flaws

**Comment by ghost** [comment]
If the info is still up to date, Netflix uses mithril or not?
Looked it up and it uses React.

**Reply by StephanHoyer** [reply]
Do we reference that somewhere? If yes, I really don't know if this is still the case, maybe for internal stuff.🤷‍♀️
